lms/djangoapps/discussion_api/api.py

```diff
@@ -40,7 +40,11 @@
     comment_voted,
     comment_deleted,
 )
-from django_comment_client.utils import get_accessible_discussion_modules, is_commentable_cohorted
+from django_comment_client.utils import (
+    get_accessible_discussion_modules,
+    get_discussion_module,
+    is_commentable_cohorted,
+)
 from lms.djangoapps.discussion_api.pagination import DiscussionAPIPagination
```

Comment:
Can you move the other `discussion_api` imports (LL19-27) down here as well, and rename them as `lms.djangoapps.discussion_api`?